tests/postgres_tests/test_hstore.py

```diff
@@ -20,6 +21,32 @@ class HStoreTestCase(PostgreSQLTestCase):
     pass
 
 
+class HstoreOIDTests(HStoreTestCase):
+
+    def test_hstore_at_most_zero_queries(self):
+        from django.contrib.postgres.signals import get_hstore_oids
+        # oids cached after app's ready() call
+        with self.assertNumQueries(0):
+            get_hstore_oids(connection.alias)
+
+    def test_citext_at_most_zero_queries(self):
+        from django.contrib.postgres.signals import get_citext_oids
+        # oids cached after app's ready() call
+        with self.assertNumQueries(0):
+            get_citext_oids(connection.alias)
+
+    def test_hstore_return_values(self):
+        from django.contrib.postgres.signals import get_hstore_oids
+        oids, array_oids = get_hstore_oids(connection.alias)
+        self.assertTrue(isinstance(oids, tuple))
```

Comment:
Use `assertIsInstance()`. Is it worth checking something like that it's a tuple integers of a certain length?

Reply:
Agreed, `assertIsInstance ()` is better choice. Checking for certain length makes very little sense.